data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+			choice
+				`	"Why don't you just go back to human space?"`
+					goto "why not go to human space"
+				`	"I'll make sure to do that."`
+			label "anna thanks you"
+			`	"Thanks again, <first> <last>." I'll forever be grateful to you for helping me. I never thought I'd be successful as an author, but when I met you, you helped me turn my career into something I thought I'd never see.`
+			choice
+				`	"Just helping out."`
+				`	"It's okay."`
+			`	With that, you leave the room, and eventually the building. You board the train, walk past the same series of endless shops and streets in the spaceport city, before returning to your ship. As soon as you're in your ship, you add a reminder to visit Anna within six months.`
+				decline
+			label "why not go to human space"
+			`	"I need to be available at all times, so I can answer their questions. Do not worry about me."`
+			choice
+				`	"Got it. Good luck publishing your book."`
+					goto "anna thanks you"
+
+event "beyond the hyperdrive is published"
+
+mission "Fangvin: Struggling Author 5A"
+	landing
+	source
+		not near "Quaru" 0 100
+	destination "Inmost Blue"
+	to offer
+		has "event: beyond the hyperdrive is published"
+		not "Fangvin: Struggling Author 5B: offered"
+	on offer
+		conversation
+			`Checking your computer feed for the latest galactic news and trade prices, you get a reminder from your computer.`
+			`	You read it, and recall that you had helped an author named Anna Fangvin get her book, 'Beyond the Hyperdrive', published about half a year ago. You should return to <destination> and check on her.`
+				accept
+	on complete
+		conversation "anna invites you to book signing"
+
+mission "Fangvin: Struggling Author 5B"
+	landing
+	source
+		near "Quaru" 0 100
+	destination "Inmost Blue"
+	to offer
+		has "event: beyond the hyperdrive is published"
+		not "Fangvin: Struggling Author 5A: offered"
+	on offer
+		conversation
+			`As you land on the spaceport, you check your messages. You see a message from Anna Fangvin, the author who you helped about half a year ago. In the message, she explains that her novel, 'Beyond the Hyperdrive', has been approved. You should return to <destination> and meet her.`
+				accept
+	on complete
+		conversation "anna invites you to book signing"
+
+conversation "anna invites you to book signing"
+	action
+		log "People" "Anna Fangvin" "Anna is an author of science fiction books. She never found success in human space, until she visited the Coalition and got her novel, 'Beyond the Hyperdrive', about a group of humans stealing jump drives from the Quarg, published. Upon publication, her book quickly became a bestseller in Kimek space."
+		log "Helped somebody named Anna Fangvin get her book published in Coalition space. Apparently, it's a bestseller now. The book was 'Beyond the Hyperdrive', something about humans stealing jump drives from the Quarg. Also found out she was ambidextrous."
+		"salary: Anna Fangvin" = 5000
+		payment 2000000
```

Comment:
The raw payment is fine, but I really don't know about the salary. First, it's way too high given existing salaries in the game; I don't doubt books could make a good bit of money daily, but considering Turner Incorporated sets up a shipyard and outfitter in Hai space, businesses where money flows a lot more than books, and the final salary for that is 4000, 5000 just isn't going to mesh well with other salaries.
Not even counting that, though, I'm just not sure it's much appropriate to add in a salary for this mission set. I think Anna expresses her appreciation for the player's help via rewards well enough with the generous base payment.